commands/api.go

```diff
@@ -67,6 +67,9 @@ var cmdApi = &Command{
 		Parse response JSON and output the data in a line-based key-value format
 		suitable for use in shell scripts.
 
+	-v, --verbose
```

Comment:
Let's not add `-v` for now. Hub already has a verbose mode:

```
HUB_VERBOSE=1 hub api ...
```

If we added a flag to enable this mode instead of the environment variable, we would have to support this flag across all commands, not just this one.